inference-engine/cmake/dependencies.cmake

```diff
@@ -125,6 +125,12 @@ if (THREADING STREQUAL "OMP")
     update_deps_cache(OMP "${OMP}" "Path to OMP root folder")
     log_rpath_from_dir(OMP "${OMP}/lib")
     debug_message(STATUS "intel_omp=" ${OMP})
+    
+    file(GLOB_RECURSE source_list "${OMP}/*${CMAKE_SHARED_LIBRARY_SUFFIX}*")
+    install(FILES ${source_list} 
+    DESTINATION "deployment_tools/inference_engine/external/omp/lib"
+    COMPONENT omp)
```

Comment:
1. `ie_cpack_add_component(omp)` first
2. Align next lines of install rules with `(`